src/audio/volume/volume.c

```diff
@@ -750,6 +766,8 @@ static int volume_copy(struct comp_dev *dev)
 
 	/* Get source, sink, number of frames etc. to process. */
 	comp_get_copy_limits_with_lock(source, sink, &c);
+	/* limit frames to be divided by 4 for alignment of 8-byte */
+	c.frames = c.frames >> 2 << 2;
```

Comment:
probably would be cleaner to mask these bits

Reply:
c.frames = c.frames & 0xFFFFFFFC; how about this way?

Reply:
Please try with profiler which one is less cycles

Reply:
you should use the `&=` operator also this might be easier to read
`c.frames &= ~0x3`

Reply:
@singalsu I tested it with xtensa simulator profiler, these two methods cost the same cycles. So I modified it like @cujomalainey suggested.  And thanks.